src/settings/MatchProfiles/MatchProfiles.js

```diff
@@ -154,6 +154,9 @@ export class MatchProfiles extends Component {
   defaultNewRecordInitialValues = {
     name: '',
     description: '',
+    incomingRecordType: 'MARC',
```

Comment:
Please add some TODO comment which says that these values are hardcoded now and will need to be changed in future.

Reply:
Will be moved to mockups untill backend will be ready to serve these for us.